include/configs/zynq-common.h

```diff
@@ -284,6 +285,9 @@
                 "fi; " \
 		"if test \"${refclk_source}\" = \"external\" && test \"${model}\" = \"Analog Devices PlutoSDR Rev.C (Z7010/AD9363)\" ; then " \
 			"fdt rm /amba/gpio@e000a000/clock_internal_en; " \
+			"if test -n \"${ad936x_ext_refclk_override}\"; then " \
```

Comment:
Can we make ad936x_ext_refclk_override also available for Rev.A and B devices?
I see no reason why not. The problem is that ad936x_ext_refclk get's always overwritten by the extra-env

Reply:
Yes I forgot about that overwrite. I can move it outside.